force-app/main/default/flexipages/Funding_Program_Record_Page.flexipage-meta.xml

```diff
@@ -6,6 +6,14 @@
                 <name>collapsed</name>
                 <value>false</value>
             </componentInstanceProperties>
+            <componentInstanceProperties>
+                <name>enableActionsConfiguration</name>
+                <value>false</value>
+            </componentInstanceProperties>
+            <componentInstanceProperties>
+                <name>hideChatterActions</name>
+                <value>false</value>
+            </componentInstanceProperties>
```

Comment:
Must be new default FlexiPage config?

Reply:
Yup, these just got added when I pulled down the org changes